CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 3.7.8
+
+## Deprecation
+
+- Started the deprecation of all functions below with alert on console:
+
+  - getBodyElementFromCurrentBrowserOrBrowserInstance
+  - openNewBrowserInTheSamePage
+  - clickWhenClickable
+  - fillFieldWithTextWhenVisible
+  - fillInputFieldWithFileWhenPresent
+  - clearFieldWhenVisible
+  - clearFieldWhenVisibleAndFillItWithText
+  - tapWhenTappable
+  - fillFieldWithTextWhenVisibleAndPressEnter
+  - scrollToElementWhenVisible
+
+- Included alert on console informing to remove the `errorMessage` argument from all wait function.
+
+- Included alert on console about 2 functions that will be removed on version 4.0.0.
```

Comment:
We don't use the past tense in the changelog.
Maybe it could be re-worded like: `Include warnings in the console about two functions that will be removed on version 4.x.x.`